tdrs-backend/gunicorn.py

```diff
@@ -0,0 +1,24 @@
+"""Gunicorn development config file."""
+
+# WSGI application path MODULE_NAME:VARIABLE_NAME
+wsgi_app = "tdpservice.wsgi:application"
+
+# The granularity of Error log outputs
+# loglevel = "debug"
+
+# The number of worker processes for handling requests
+workers = 2
+# The socket to bind
+bind = "0.0.0.0:8080"
+# Restart workers when code changes (development only!)
+reload = False
+# Write access and error info to /var/log
+# accesslog = errorlog = "/var/log/gunicorn/dev.log"
```

Comment:
Hmm, I wonder if we *should* write a log to disk? I think we have something about Logstash or Grafana in the devsecops roadmap.

Reply:
I think we should keep it simple for now until we have log server or monitoring software. We can discuss the logging strategy.